### PR TITLE
feat: add arm support

### LINKED ISSUE #164: Support ARM architecture
When running analysis on ARM arch, analysis fails [here](https://github.com/editorconfig-checker/editorconfig-checker.php/blob/master/src/EditorconfigChecker/Utilities.php#L20). Can you add support for it? `php_uname('m')` returns `aarch64` on ARM.

### PR DESCRIPTION
**Pull request by mstruebing**
Closes #164

### CODE REVIEW

File: src/EditorconfigChecker/Utilities.php [hunk 1]
```diff
@@ -11,17 +11,17 @@ public static function getCurrentArch(): string
...
-            // TODO: Whats here?
-            printf('ERROR: Unexpected, please contact the maintainer or provide a pull request :)%s', PHP_EOL);
-            exit(1);
+        switch($arch) {
+            case 'x86_64':
+                return "amd64";
+            case 'i386':
+                return "386";
+            case 'aarch64'
+                return "arm64"
```
Comment (janedbal) [inline src/EditorconfigChecker/Utilities.php:20]: semicolon missing

Comment (mstruebing) [reply src/EditorconfigChecker/Utilities.php]: Semicolon added 🚀

File: src/EditorconfigChecker/Utilities.php [hunk 2]
```diff
@@ -11,17 +11,17 @@ public static function getCurrentArch(): string
...
-        } else {
-            // TODO: Whats here?
-            printf('ERROR: Unexpected, please contact the maintainer or provide a pull request :)%s', PHP_EOL);
-            exit(1);
+        switch($arch) {
+            case 'x86_64':
+                return "amd64";
+            case 'i386':
+                return "386";
+            case 'aarch64'
```
Comment (janedbal) [inline src/EditorconfigChecker/Utilities.php:19]: colon missing..

Seems like some basic CI would be beneficial here :)

**Comment by mstruebing** [comment]
Could you test if this actually works and runs on your machine with the correct `ec`-binary from https://github.com/editorconfig-checker/editorconfig-checker

**Comment by janedbal** [comment]
Seems like it is working fine

**Pull request merged**